lib/tenantmetrics/counter_map_test.go

```diff
@@ -88,3 +90,34 @@ func TestCounterMapConcurrent(t *testing.T) {
 		t.Fatalf("unexpected counter value; got %d; want %d", n, concurrency*10*5)
 	}
 }
+
+func BenchmarkCounterMapGrowth(b *testing.B) {
```

Comment:
Please move the benchmark to the counter_map_timing_test.go (it does not exist yet, you'd need to create it). This is a convention we use: benchmarks are placed into _timing_test.go files.

Reply:
Also, we've been using so called f-tests instead of table-driven tests. See: https://itnext.io/f-tests-as-a-replacement-for-table-driven-tests-in-go-8814a8b19e9e

May I ask to you to re-structure the test in order to adhere to our coding style? It should be something like this:

```go
func BenchmarkCounterMapGrowth(b *testing.B) {
        f := func(b *testing.B, n uint32, nProcs int) {
                b.Helper()

                for i := 0; i < b.N; i++ {
                        cm := NewCounterMap("foobar")
                        var wg sync.WaitGroup
                        for range nProcs {
                                wg.Add(1)
                                go func() {
                                        for i := range numTenants {
                                                cm.Get(&auth.Token{AccountID: i, ProjectID: i}).Inc()
                                        }
                                        wg.Done()
                                }()
                        }
                        wg.Wait()
                }
        }

        b.Run("n=100,nProcs=GOMAXPROCS", func(b *testing) {
                f(b, 100, runtime.GOMAXPROCS(0))
        })

        b.Run("n=100,nProcs=2", func(b *testing) {
                f(b, 100, 2)
        })

        b.Run("n=1000,nProcs=2", func(b *testing) {
                f(b, 1000, 2)
        })

        b.Run("n=10000,nProcs=2", func(b *testing) {
                f(b, 10000, 2)
        })
}
```